### PR TITLE
UICAL-55 - Check correct using of time zone for calendar

### PR DESCRIPTION
**Pull request by maximdidenkoepam**
# Purpose
Check that correct format used for datepickers and timepickers in calendar module.

# Link
https://issues.folio.org/browse/UICAL-55

### CODE REVIEW

File: settings/OpeningPeriodForm/InputFields.js [hunk 1]
```diff
@@ -71,6 +71,9 @@ class InputFields extends React.Component {
           label={<FormattedMessage id="ui-calendar.validFrom" />}
           onChange={this.setStartDate}
           required
+          timeZone="UTC"
+          backendDateStandard="YYYY-MM-DD"
+          dateFormat="MM/DD/YYYY"
```
Comment (zburke) [inline settings/OpeningPeriodForm/InputFields.js]: I confess I'm not familiar with this module, but I'm surprised to see a locale-specific timezone, `MM/DD/YYYY`, here. Are you certain hard-coding a specific date format is the correct solution here?

**Review by zburke — COMMENTED** [review]
The use of UTC and local timezones, as well as YYYY-MM-DD and locale-specific formats, is confusing. This isn't a problem with the code, it's an inherent problem of dealing with dates and conversions. 

I think we need some comments in the code explaining why each timezone and format is used in each place, and an overall explanation of the conversions going on. Does that make sense?

File: settings/OpeningPeriodForm/InputFields.js (hunk 1 above)
Comment (maximdidenkoepam) [reply settings/OpeningPeriodForm/InputFields.js]: I used format which is used among the whole module, but without receiving it from translations.

[Component](https://github.com/folio-org/ui-calendar/blob/cac1ee26da4745b4d38bbb1a4e4ec183bed81e06/settings/ServicePointDetails.js#L105)

[Translations](https://github.com/folio-org/ui-calendar/blob/cac1ee26da4745b4d38bbb1a4e4ec183bed81e06/translations/ui-calendar/en.json#L72)

Or you want me to do the same? Did you mean that for each translation can be different format?

Comment (zburke) [reply settings/OpeningPeriodForm/InputFields.js]: Isn't date-format is locale-dependent? i.e. in the en-US locale people expect to see mm/dd/yyyy, in sp they expect dd/mm/yyyy, in de dd.mm.yyyy etc.

**Review by zburke — APPROVED** [review]

**Pull request merged**